docsets/Agora_Recording_CPP/docset.json

```diff
@@ -0,0 +1,12 @@
+
+{
+    "name": "Agora C++",
+    "version": "2.3.3",
+    "archive": "Agora_CPP.tgz",
```

Comment:
The archive you added is called "Agora_Recording_CPP.tgz".

Reply:
Yes. The newly-added API reference is for a different product.

Reply:
Sorry, I've already updated the information.